WebDriverManager/DriverConfigs/Impl/ChromeConfig.cs

```diff
@@ -63,5 +66,25 @@ public virtual string GetLatestVersion()
                 }
             }
         }
+
+        public virtual string GetMatchingBrowserInstalledVersion()
+        {
+            string chromeVersionInstalled = RegistryHelper.GetInstalledBrowserVersion(BrowserExecutableFileName);
+            chromeVersionInstalled = VersionHelper.GetVersionSkippingRevision(chromeVersionInstalled);
+            var uri = new Uri(DriverQueryVersionUrl.Replace("<version>", chromeVersionInstalled));
+            var webRequest = WebRequest.Create(uri);
+            using (var response = webRequest.GetResponse())
+            {
+                using (var content = response.GetResponseStream())
+                {
+                    if (content == null) throw new ArgumentNullException($"Can't get content from URL: {uri}");
+                    using (var reader = new StreamReader(content))
+                    {
+                        var version = reader.ReadToEnd().Trim();
+                        return version;
+                    }
+                }
+            }
```

Comment:
You can merge these `using`s (omit the brackets of the first one) or if OK in this project also use the new `using var` notation.

Reply:
In my opinion current implementation is pretty simple and obvious

Reply:
I'll let you guys decide this issue. I grabbed already existing code to implement this part.